zookeeper-docs/src/documentation/content/xdocs/zookeeperAdmin.xml

```diff
@@ -764,6 +764,18 @@ server.3=zoo3:2888:3888</programlisting>
             </listitem>
           </varlistentry>
 
+          <varlistentry>
+            <term>observerMasterPort</term>
```

Comment:
If we leave `observerMasterPort` as a separation configuration option (as opposed to part of server:ports configuration), then we will not be able to reconfigure this port using dynamic reconfig, right? Is this expected?

Reply:
That is correct, we will not be able to change this with a reconfig. I did not want to alter the format of the reconfig operation in this change since I felt it was significant enough to warrant its own patch. A more extensible configuration format would also benefit https://issues.apache.org/jira/browse/ZOOKEEPER-3166.